modules/tinymce/src/plugins/link/main/ts/core/Utils.ts

```diff
@@ -78,6 +78,9 @@ const getAnchorElement = (editor: Editor, selectedElm?: Element): Optional<HTMLA
   }
 };
 
+const isInAnchor = (editor: Editor, selectedElm?: Element): boolean =>
```

Comment:
Can you also update the uses in `Actions.ts` as well to remove this duplication please?

- [ ] Replace all uses of `getAnchorElement(editor, selectedElm).isSome()` with `isInAnchor`

Reply:
Done!